package.json

```diff
@@ -43,5 +44,12 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
```

Comment:
Notice that all the linting and formatting dependencies are `"devDependencies"`, that means they only will bundle with your local development environment, not deployed to a production environment.